docs_app/content/guide/subject.md

```diff
@@ -0,0 +1,364 @@
+# Subject
+
+**What is a Subject?** An RxJS Subject is a special type of Observable that allows values to be multicasted to many Observers. While plain Observables are unicast (each subscribed Observer owns an independent execution of the Observable), Subjects are multicast.
+
+<span class="informal">A Subject is like an Observable, but can multicast to many Observers. Subjects are like EventEmitters: they maintain a registry of many listeners.</span>
```

Comment:
I have wondered what is the purpose of this span class used for?

Reply:
I found it. It is supposed to be used as the blockquote but the new docs does not have the css in place to show it that way.